build/common.props

```diff
@@ -9,7 +9,7 @@
   </ItemGroup>
   <!-- When building on non-Windows machines, we need the reference assemblies to build full-framework assemblies. -->
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' AND '$(DotNetBuildFromSource)' == 'true'">
-     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-003" PrivateAssets="All" />
+     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-5" PrivateAssets="All" />
```

Comment:
Is there a property that we can set here? 
That way source-build can just upgrade on their without an action from us. Similar to what we're doing with the MSBuild deps.

Reply:
There's a property defined [here](https://github.com/dotnet/arcade-services/blob/95a3b49351e5612e6fe2bb8799d98368bf66c0ba/eng/Versions.props#L10) but, it seems that it doesn't get injected in the source-build properties file (Repo API).

Reply:
I see. 
If there are plans to get it injected, please set that property similarly to what we've done with the MSBuild deps. 

I'm fine with the change as is otherwise.